server/game/cards/01-Core/MeddlingMediator.js

```diff
@@ -6,7 +6,7 @@ class MeddlingMediator extends DrawCard {
         this.action({
             title: 'Take 1 fate or 1 honor',
             phase: Phases.Conflict,
-            condition: context => this.game.getConflicts(context.player.opponent).filter(conflict => !conflict.passed).length > 1,
+            condition: context => context.player.opponent && this.game.getConflicts(context.player.opponent).filter(conflict => !conflict.passed).length > 1,
             target: {
                 mode: TargetModes.Select,
```

Comment:
Similar to previous comment. Don't solve this in implementation but in getConflicts() by return an empty array